### DISCUSSION TITLE
can't reapply

### DISCUSSION THREAD

**Discussion by maxdnc** [community/community#59776] [GitHub Education]
### Hi everyone,

Hello, so I'm trying to reapply since I'm still a remote student, but for now I have this error :

**We require applicants of OpenClassrooms to use one of these school-issued email addresses to apply:
- openclassrooms.com
- itopi.fr
Please select a different email address.**

but I can't put this as an email on my profile i can't verify the mail ? 
the first apply was an easy process  and i could put my personal mail adress
Any solution ? thanks

**Comment by ndecr** [comment]
Same problem here. I wanted to renew for the year as I'm still a student, and I was also denied by the form even though I didn't have any issues last year. Have you found a solution since your message by any chance?

**Reply by maxdnc** [reply]
Nothing, they ignore the problem...

**Reply by VTJ-Dumay-Pothuon** [reply]
I reached for Openclassrooms directly, and they pretend they are contacting Github about it. It was two weeks ago, OC closed my request, and now my Github Academy ticket is expired. So I would say they do ignore the problem… on both sides.

**Comment by nonvenomous** [comment]
I have a quite similar issue.
There are more people with this issue.
A bigger thread here https://github.com/orgs/community/discussions/59360

**Comment by samus-aran** [comment]
Hi @maxdnc 👋 

We're sorry you're experiencing this. We have updated this school's information. Can we please ask you to try now and re-apply. We've raised this experience with our internal team but we hope this helps you proceed to re-verify your academic status. 👍
